src/main/java/com/dev/cinema/dao/impl/OrderDaoImpl.java

```diff
@@ -0,0 +1,60 @@
+package com.dev.cinema.dao.impl;
+
+import com.dev.cinema.dao.OrderDao;
+import com.dev.cinema.exception.DataProcessingException;
+import com.dev.cinema.lib.Dao;
+import com.dev.cinema.model.Orders;
+import com.dev.cinema.model.User;
+import com.dev.cinema.util.HibernateUtil;
+import java.util.List;
+import javax.persistence.criteria.CriteriaBuilder;
+import javax.persistence.criteria.CriteriaQuery;
+import javax.persistence.criteria.JoinType;
+import javax.persistence.criteria.Predicate;
+import javax.persistence.criteria.Root;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+
+@Dao
+public class OrderDaoImpl implements OrderDao {
+    @Override
+    public Orders create(Orders order) {
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            session.save(order);
+            transaction.commit();
+            return order;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException(
+                    "Cant create order with description " + order.toString()
```

Comment:
```suggestion
                    "Cant create order with description " + order
```
toString() is called implicitly